helm/hookshot/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: v2
+name: hookshot
+description: Deploy a Matrix Hookshot instance to Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.13
```

Comment:
It would be good for this to follow the version in package.json somehow so it's one less thing to remember, might be another script?